### PR TITLE
Add server API

### PR DESCRIPTION
**Pull request by bionanek**

### CODE REVIEW

**Comment by bionanek** [comment]
* **#6** <a href="https://app.graphite.dev/github/pr/bionanek/togather2/6?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bionanek and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by bionanek** [comment]
test

File: graphite-demo/server.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+// Fake data for tasks
+const tasks = [
+  {
+    id: 1,
+    description: 'Complete monthly financial report'
```
Comment (kachmashk) [inline graphite-demo/server.js:9]: lgtm

Comment (bionanek) [reply graphite-demo/server.js:9]: ok

**Pull request closed without merging**